### ISSUE TITLE
Doesn't work in Safari

### ISSUE THREAD

**Issue by panmona** [Hivenfour/SeamlessLoop#8]
I got this library to work in Chrome but not in Safari.
I played around a bit with the play-/stopDelay but the delay was still noticeable in the end.

**Comment by fluffy-critter** [comment]
In Safari I have the opposite problem, it seems to restart the loop early most of the time.

In my experience Safari is very difficult to work with in terms of the `<audio>` element and for now I think I'll just continue to use a regular audio with a default of ogg (which works most places nowadays and is gapless), and just not worry about gapless looping on Safari.

**Comment by panmona** [comment]
Is that so with ogg? Never heard of that.

Btw I solved my problem with this: https://stackoverflow.com/questions/46926033/create-seamless-loop-of-audio-web

**Comment by fluffy-critter** [comment]
Yeah, ogg supports gapless looping, and works fine in Firefox and Chrome. 

I'll look into the Web Audio API, thanks!

**Comment by panmona** [comment]
Huh okay. I used wav for gapless looping in the Web Audio API.
Are ogg files smaller?

**Comment by fluffy-critter** [comment]
Yes, org files are way, way smaller. They’re about the same compression efficiency as mp3. I don’t know if they work with WebAudio though, but they’re also pretty easy to decode so I bet there’s a pure-JavaScript decoder out there for browsers that don’t support it directly.
